storage/client_raft_test.go

```diff
@@ -541,6 +541,119 @@ func TestStoreRangeReplicate(t *testing.T) {
 	}
 }
 
+// TestStoreRangeDownReplicate verifies that the replication queue will notice
+// over-replicated ranges and remove replicas from them.
+func TestStoreRangeDownReplicate(t *testing.T) {
+	defer leaktest.AfterTest(t)
+	mtc := startMultiTestContext(t, 5)
+	defer mtc.Stop()
+
+	store1 := mtc.stores[0]
```

Comment:
I would call this `store0` (use index in the stores array instead of the store ID). Neither way is ideal, but I've found it's less confusing overall to use the index in the test code, since that's what you have to pass to methods like `replicateRange`.